### DISCUSSION TITLE
Upcoming minor release 6.2

### DISCUSSION THREAD

**Discussion by jamesdaniels** [angular/angularfire#2712] [Roadmap]
**This is a placeholder for the discussion of the upcoming 6.2 release candidates**.

The goals of the 6.2 release are to:

* provide lazy versions of Firestore, Database, and Storage
* add experimental support  for server-side cookie Authentication, Analytics, and Remote Config
* better state-transfer support
* support for using the Firebase memory-only build (`firebase/memory`)
* pre-rendering, hosting channels, and custom functions for `ng deploy`
* and more...

### More information

- [Public Roadmap card](https://github.com/angular/angularfire/projects/5#card-50953529)
- [Milestone](https://github.com/angular/angularfire/milestone/22)

**Comment by rbogdy** [comment]
Any news on this minor update?

**Reply by jamesdaniels** [reply]
We ended up not having bandwidth for another 6.x release before the new Firebase JS SDK (v9) and AF v7.0 went to GA. Many of these features have landed in 7.0 or will be landing shortly in 7.1.

**Comment by markgoho** [comment]
could we add angular v12 support in this?

**Reply by jamesdaniels** [reply]
Addressed in `6.1.5`.

**Comment by imbradyboy** [comment]
Will this include support for the storage emulator?

**Reply by jamesdaniels** [reply]
Added in `7.0`
